CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.11.0
```

Comment:
Oh, I thought there were some interesting changes to be released based on the commits but apparently not (looks like a lot of bug fixes), never mind then.

Reply:
This is an interesting change, but it's still more of a bugfix, I didn't write any release notes for it: https://github.com/packit/specfile/pull/153